runtime/pointers.ts

```diff
@@ -3045,22 +3056,33 @@ export class Pointer<T = any> extends Ref<T> {
      */
     public static async cleanupSubscribers() {
         logger.debug("cleaning up subscribers");
-        const onlineCache = new Map<Endpoint, boolean>();
         let removeCount = 0;
-        
-        for (const pointer of Pointer.iterateAllPointers()) {
-            for (const subscriber of pointer.subscribers) {
-                const isOnline = onlineCache.get(subscriber) ?? await subscriber.isOnline();
-                onlineCache.set(subscriber, isOnline);
-                if (!isOnline) {
-                    pointer.removeSubscriber(subscriber);
-                    removeCount++;
-                }
+
+        for (const [endpoint, pointers] of Pointer.#endpoint_subscriptions) {
+            if (await endpoint.isOnline()) continue;
```

Comment:
Maybe we should consider running the cleanup in parallel instead of having a one-by-one loop here.
Cleaning up many offline subscribers could get pretty time intensitive since we need to wait for the timeout of the online state check for all of the subscribers one by one.

Reply:
Im not sure about this. I did this in sequence on purpose to keep the DATEX channels free to send other messages. If we check the online state for hundreds of endpoints at once, why might overstress the network and/or communication channels.
Since this is not really a time sensitive issue, its not that big of a problem if it takes some time until all endpoints are cleaned up.
What do you think=

Reply:
Got your point. I think that overstressing the network should't occur at any point, otherwise, scaling will be a very fun experience. Maybe we can think of introducing some sort of networking queue with priority levels? (Not just for this comment in here, I'm quite sure there a tons of usages where we might benefit by using such a system)
In this regard our network and clients should not get blocked if we are sending ~10 requests as bundle, wait until all requests are done or failed and continue with the next bundle.
I think that pretty much all communication channels for web are able to handle quite a bunch of messages in parallel, especially since the messages for online check in this case should be really small.
I just wanted to add some unsorted thoughts on this topic but I'm closing this for now!